src/Games/NexusMods.Games.StardewValley/Models/SMAPIManifestLoadoutFile.cs

```diff
@@ -9,5 +13,17 @@ public partial class SMAPIManifestLoadoutFile : IModelDefinition
 {
     private const string Namespace = "NexusMods.StardewValley.SMAPIManifestLoadoutFile";
 
-    public static readonly MarkerAttribute ManifestFile = new(Namespace, nameof(ManifestFile));
+    public static readonly MarkerAttribute ManifestFile = new(Namespace, nameof(ManifestFile)) { IsIndexed = true };
+
+    public static IEnumerable<ReadOnly> GetAllInLoadout(IDb db, LoadoutId loadoutId, bool onlyEnabled)
+    {
+        var entityIds = db.Datoms(
+            (ManifestFile, Null.Instance),
+            (LoadoutItem.Loadout, loadoutId)
+        );
+
+        return entityIds
+            .Select(entityId => Load(db, entityId))
+            .Where(loadoutItem => loadoutItem.IsValid() && (!onlyEnabled || onlyEnabled && loadoutItem.AsLoadoutFile().AsLoadoutItemWithTargetPath().AsLoadoutItem().IsEnabled()));
```

Comment:
I'm wondering whether loading the `LoadoutItem` from the `EntityId` is faster than going through the chain. Hopefully it doesn't matter.